robottelo/cli/host.py

```diff
@@ -460,3 +460,27 @@ def create(cls, options=None):
         """Create new network interface for host"""
         cls.command_sub = 'create'
         cls.execute(cls._construct_command(options), output_format='csv')
+
+
+class HostTraces(Base):
+    """Manages traces on your hosts
+
+    Usage::
+        hammer host traces [OPTIONS] SUBCOMMAND [ARG] ...
+
+    Subcommands::
+        list                          List services that need restarting on the host
+        resolve                       Resolve Traces
+    """
+
+    command_base = 'host traces'
+
+    @classmethod
+    def list(cls, options=None):
+        cls.command_sub = 'list'
```

Comment:
Please add documentation for each command.
```suggestion
    def list(cls, options=None):
        """List services that need restarting on the host
        Usage:
            hammer host traces list [OPTIONS]
        Options:
         --host[-id]                   Set the current host context for the request.
                                       Name/Id can be used
         -h, --help                    Print help
        """
        cls.command_sub = 'list'
```